trip_data.py

```diff
@@ -0,0 +1,69 @@
+import csv
+import datetime # used for dates
+from dataclasses import dataclass
+from typing import List
+
+@dataclass
+class TripData: # main class with data read from csv file
+
+   duration: int
+   start_date: datetime
+   end_date: datetime
+   start_st_number: int
+   start_station: str
+   end_station_number: int
+   end_station: str
+   bike_number: str
+   member_type: str
+
+   def __init__(self): # builder
+      pass
+
+class FileReader:
```

Comment:
FileReader has nothing to do with trip_data. Sounds like a candidate for a separate file